docs/software/file_servers.md

```diff
@@ -239,37 +239,71 @@ Network file system server.
     exportfs -ra
     ```
 
-    Alternatively you can restart the service (`systemctl restart nfs-kernel-server`).
+    Alternatively the service can be restarted (`systemctl restart nfs-kernel-server`).
 
     The actual access configuration can be displayed with the command
 
     ```
     exportfs
     ```
 
-    On the client side you can query the mountable exports with the command
+    On the client side the mountable exports can be queried with the command
 
     ```
     showmount -e <NFS_SERVER>
     ```
 
-=== "Default configuration / increase security"
+=== "Default configuration / limit access"
 
     By default the DietPi NFS installation exports the directory `/mnt/dietpi_userdata` for everyone. This is configured in `/etc/exports.d/dietpi.exports`. You can edit this file to restrict the access.
 
-    E.g. you could limit the access to the NFS share by setting a IP address range:
+    E.g. the access to the NFS share can be limited by setting a IP address range with these steps:
 
-    - Edit the following file: `/etc/exports.d/dietpi.exports`
-    - To only allow users access with an IP address range of 192.168.0.1-255
+    - Editing the file `/etc/exports.d/dietpi.exports`
+    - Changing the file to only allow users access e.g. with an IP address range of 192.168.0.1-255 
 
         ```
         /mnt/dietpi_userdata 192.168.0.*(rw,async,no_root_squash,fsid=0,crossmnt,no_subtree_check)
         ```
 
-    - Activate the new configuration (`systemctl restart nfs-kernel-server` or `exportfs -ra`)
+    - Activating the new configuration (`systemctl restart nfs-kernel-server` or `exportfs -ra`)
+
+=== "NFS v3 disable/enable"
+
+    One option to disable NFS v3 is to add a file to the directory `/etc/nfs.conf.d/` within the section `[nfsd]`:
```

Comment:
```suggestion
    One option to disable NFS v3 is to add a file to the directory `/etc/nfs.conf.d/` with the following content:
```
Somehow it sounds like the file should be added to the `[nfsd]` section, instead of that it needs to contain an `[nfsd]` section. But As the content is shown below, I think it does not need to be explicitly mentioned in the text.